docs/migrations/01207.md

```diff
@@ -70,8 +70,25 @@ const MyStyledComponent =
   });
 ```
 
-All usage of `$ref` has been removed. You can now move from `$ref` to `ref` and you will see a warning
-in the console describing the change.
+We understand that adding type checks to your code all of a sudden may introduce a substantial
+migration effort. Because of this, you can run a codemod to add `// $FlowFixMe` comment above all
+uses of baseui's `styled` function. Doing so will leave your code in the same state as before, no
+type security, with the benefit of migrating type checks at your leisure. Either update the types
+along with the baseui version update or run the codemod to save the changes for later. Follow the
+instructions below to run the codemod locally.
+
+```
+npm install -g jscodeshift
+jscodeshift -t node_modules/baseui/codemods/styled-flowfixme.js <transform-path>
+```
+
+Use the -d option for a dry-run and use -p to print the output for comparison.
+
+## $ref => ref
```

Comment:
I think you have to use H3 here, as `## Styletron update to v5` is an H2

Reply:
```suggestion
### $ref => ref
```